script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -1097,7 +1098,7 @@ Panting heavily, she releases her grip on the halberd.
 -- 力任せに暴れすぎた結果だ。もともと彼女には槍斧を振るうだけの筋力はない。槍斧に宿った『軽量化』の秘蹟を一撃ごとに使わなければ有効打にはなりえない。
 It's only natural she'd be exhausted after flailing it about as hard as she did.
  She doesn't actually possess the strength necessary to wield it in the first place.
- She'd never be able to hit anything with it if she didn't activate the sacrament of 'Weight Reduction' it carries for every swing.
+ She'd never be able to hit anything with it if she didn't activate the sacrament of %{i}Weight Reduction%{/i} it carries for every swing.
```

Comment:
Dunno if this is really a lore term so much as just a name

Reply:
Yeah, it's the name of her spell.
We could probably just leave it uncaptialized.